frame/gilt/src/lib.rs

```diff
@@ -488,7 +493,40 @@ pub mod pallet {
 		}
 	}
 
+	/// Issuance information returned by `issuance()`.
+	pub struct IssuanceInfo<Balance> {
+		/// The balance held in reserve over all active gilts.
+		pub reserved: Balance,
+		/// The issuance not held in reserve for active gilts. Together with `reserved` this sums to
+		/// `Currency::total_issuance`.
+		pub non_gilt: Balance,
+		/// The balance that `reserved` is effectively worth, at present. This is not issued funds
+		/// and could be less than `reserved` (though in most cases should be greater).
+		pub effective: Balance,
+	}
+
 	impl<T: Config> Pallet<T> {
+		/// Get the target amount of Gilts that we're aiming for.
+		pub fn target() -> Perquintill {
+			ActiveTotal::<T>::get().target
+		}
+
+		/// Returns information on the issuance of gilts.
+		pub fn issuance() -> IssuanceInfo<BalanceOf<T>> {
+			let totals = ActiveTotal::<T>::get();
+
+			let total_issuance = T::Currency::total_issuance();
+			let nongilt_issuance = total_issuance.saturating_sub(totals.frozen);
+			let effective_issuance = totals.proportion.left_from_one()
+				.saturating_reciprocal_mul(nongilt_issuance);
+
+			IssuanceInfo {
+				reserved: totals.frozen,
+				non_gilt: nongilt_issuance,
+				effective: effective_issuance,
+			}
```

Comment:
```suggestion
			let non_gilt = total_issuance.saturating_sub(totals.frozen);
			let effective = totals.proportion.left_from_one()
				.saturating_reciprocal_mul(nongilt_issuance);

			IssuanceInfo {
				reserved: totals.frozen,
				non_gilt,
				effective,
			}
```

Looks nicer but ignore if you want